wizard/WizardDaemonSettings.qml

```diff
@@ -92,6 +92,34 @@ ColumnLayout {
                     blockchainFolder.focus = true;
                 }
             }
+            onTextUpdated: {
+                var dbExists = daemonManager ? daemonManager.checkLmdbExists(blockchainFolder.text) : false
+                existingDbWarning.visible = dbExists
+                if (dbExists)
+                    pruneBlockchainCheckBox.checked = false;
+            }
+        }
+
+        RowLayout {
+            id: pruningOptionRow
+            MoneroComponents.CheckBox {
+                id: pruneBlockchainCheckBox
+                checked: !existingDbWarning.visible ? persistentSettings.pruneBlockchain : false
+                enabled: !existingDbWarning.visible
+                onClicked: {
+                    persistentSettings.pruneBlockchain =  !persistentSettings.pruneBlockchain
+                    this.checked = persistentSettings.pruneBlockchain
+                }
+                text: qsTr("Prune blockchain") + translationManager.emptyString
+            }
+
+            Text {
+                id: existingDbWarning
+                text: "A blockchain database already exists here. Select a new location to start a pruned node"
+                visible: daemonManager ? daemonManager.checkLmdbExists(blockchainFolder.text) : false
+                color: MoneroComponents.Style.defaultFontColor
+                font.family: MoneroComponents.Style.fontRegular.name
+            }
```

Comment:
Showing the warning using `appWindow.showStatusMessage` on `pruneBlockchainCheckBox` click will probably look better.

Reply:
mmmm i don't think so. I tried that and it honestly looked horrible. plus then you have to enable the check box even if you dont really want the user to click it